### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (#list_limit)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#61]
#### Issue
![image](https://user-images.githubusercontent.com/1296369/69328210-bd378d80-0c46-11ea-876b-f5cbf5c79ce8.png)

`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.3/label?application=msftAI))
#### Target application
[Templates: Styles (Site) - kh - Administration](http://localhost/kh/administrator/index.php?option=com_templates&view=styles)
#### Element path
#list_limit
#### Snippet
`<select id="list_limit" name="list[limit]" class="custom-select" onchange="this.form.submit();">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 78.0.3904.97
